### DISCUSSION TITLE
Posting Bills to Sage50

### DISCUSSION THREAD

**Discussion by mcclowes** [codatio/codat-docs#61] [Q&A] [answered]
Anyone get the following error when posting a bill to Sage Business Cloud Accounting?

```json
{
"timeoutInMinutes": 1,
"status": "Failed",
"errorMessage": "An error occurred in a downstream service. Correlation ID: c916d925b6e0bfd2f81fbf14fde22a64. Message: Push failed for Bill: see validation for more information",
"validation": {
"errors": [
{
"itemId": "line[0].AccountRef.Id",
"message": "Failed to push to Bill as line[0].AccountRef.Id is not valid as Bill (Purchase-Invoice) is not enabled in the Account Visibility settings.",
"validatorName": "Bill"
}
],
"warnings": []
},
"statusCode": 400
}
```

**Comment by dcoplowe** [comment, marked as answer]
Hi Max, 

I think we have found what's causing this problem. Any account you want to link to a line item must have visibility enabled in Sage Business Cloud. This can be found under Reporting, more..., Chart of Accounts. Here you will see a list of nominal accounts along with their visibility. Click on the account you want to allow posting to and you will see a visibility section (as shown below). Select what visibility you want to enable on the account (Purchases for bills) and you should then be able to post the bill successfully.

![image](https://user-images.githubusercontent.com/11899767/217802860-f1b11d3a-e038-45ca-bd41-d77a74396aaf.png) 

Let me know how you get on. 

Many thanks,

David
